Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+RUN apt-get -qq update && apt-get -qqy install python-pip python-dev
+RUN pip install junebug
+RUN mkdir logs
+
+EXPOSE 8080
+ENTRYPOINT ["jb", "--config=/app/example-config.yaml"]
```

Comment:
The reason why I didn't want to set the current working directory to `/app` is that we're mapping the folder as a volume, which would mean that we need a `logs` folder, and I didn't want that folder to be created on the host.